sources/packages/web/src/views/aest/CASInvoices.vue

```diff
@@ -0,0 +1,272 @@
+<template>
+  <full-page-container :full-width="true">
+    <template #header
+      ><header-navigator
+        title="Corporate Accounting Services"
+        subTitle="Invoices"
+      />
+    </template>
+    <body-header-container>
+      <template #header>
+        <body-header
+          title="Accounts payable invoicing"
+          subTitle="Please see below the list of invoices batches."
+          :recordsCount="paginatedBatches.count"
+        >
+          <template #actions>
+            <v-btn-toggle
+              mandatory
+              v-model="approvalStatusFilter"
+              class="float-right btn-toggle"
+              selected-class="selected-btn-toggle"
+              @update:model-value="filterByApprovalStatus"
+            >
+              <v-btn
+                rounded="xl"
+                color="primary"
+                :value="ApprovalStatusFilter.All"
+                class="mr-2"
+                >All statuses</v-btn
+              >
+              <v-btn
+                v-for="approvalStatus in CASInvoiceBatchApprovalStatus"
+                :key="approvalStatus"
+                rounded="xl"
+                color="primary"
+                :value="approvalStatus"
+                class="mr-2"
+                >Only {{ approvalStatus }}</v-btn
```

Comment:
Following our existing and standard UI patterns, I strongly recommend not to prefix with only. 
![image](https://github.com/user-attachments/assets/19009730-5025-4278-adc2-86f0ae5ca4db)